powerhome/playbook
# [PLAY-184] Tooltip - React Kit

#### Screens
![Screen Shot 2022-08-10 at 3 04 34 PM](https://user-images.githubusercontent.com/8194056/183984786-c3cab4f9-d2f0-4a59-92a4-5a47be4e38d5.png)
![Screen Shot 2022-08-10 at 3 04 53 PM](https://user-images.githubusercontent.com/8194056/183984867-71a8f426-0c6f-45a6-8682-6ba8cdb82a8f.png)
![Screen Shot 2022-08-10 at 3 05 05 PM](https://user-images.githubusercontent.com/8194056/183984902-7c05561a-7da6-4d99-b690-cc97467aca4f.png)
![Screen Shot 2022-08-10 at 3 05 19 PM](https://user-images.githubusercontent.com/8194056/183984945-b8acc23e-cf83-43d0-b3cf-6b52b38b3cd9.png)


#### Breaking Changes

No breaking changes, new tooltip kit for react

#### Runway Ticket URL

[[PLAY-184]](https://nitro.powerhrg.com/runway/backlog_items/PLAY-184)

#### How to test this

Check runway ticket for a complete list of all requirements for the component. Make sure to test in mobile and desktop resolutions. 

#### Checklist:
